### ISSUE TITLE
Layouts: Section component for Standard state

### ISSUE THREAD

**Issue by a8ck3n** [Automattic/jetpack#26638]
The Standard state meaning we are not waiting on indexing. The layout should include the following:

* Plan Summary component (coming in [#26054](https://github.com/Automattic/jetpack/issues/26054))
* Two `DonutMeterContainer` components showing usage
* Explanatory text with link to documentation

![Image](https://user-images.githubusercontent.com/40267301/193779896-51137887-afde-48f0-abc8-342fe80e349a.png)

If we decide to hide the RecordMeter behind a toggle, that should be a follow-up issue.
